utils/bandwidthd-php/files/config.conf

```diff
@@ -0,0 +1,7 @@
+<?php
+define("DFLT_WIDTH", 900);
+define("DFLT_HEIGHT", 256);
+define("DFLT_INTERVAL", INT_DAILY);
+
+$db_connect_string = "host=192.168.1.1 user=postgres dbname=bandwidthd"
```

Comment:
127.0.0.1 make more sense here?
also this config is lost on upgrade
https://wiki.openwrt.org/doc/devel/packages#packageconffiles_optional